db/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./Controllers/topics.controller')
+
+app.use(express.json())
```

Comment:
Are you using this yet? We should avoid adding in boilerplate code that is not required yet 👀